src/operators/Isovolume/avtIsovolumeFilter.C

```diff
@@ -640,26 +650,90 @@ avtIsovolumeFilter::ExecuteData_VTKM(avtDataRepresentation *in_dr,
                                      std::vector<double> bounds,
                                      std::vector<bool> clips)
 {
-#ifndef HAVE_LIBVTKH
+#ifndef HAVE_LIBVTKM
     return NULL;
 #else
-    int timerHandle = visitTimer->StartTimer();
-    vtkh::DataSet *in_ds = in_dr->GetDataVTKm();
-    if (!in_ds || in_ds->GetNumberOfDomains() != 1)
+    //
+    // Get the VTKM data set, the domain number, and the label.
+    //
+    avtVtkmDataSet *in_ds = in_dr->GetDataVTKm();
+    int domain = in_dr->GetDomain();
+    std::string label = in_dr->GetLabel();
+
+    if (!in_ds)
+    {
         return NULL;
+    }
 
     std::string isoVar(activeVariable != NULL ? activeVariable
                                               : pipelineVariable);
 
-    vtkh::IsoVolume iso;
+    vtkm::cont::DataSet dataset = in_ds->ds;
+
+    int timerHandle = visitTimer->StartTimer();
+
+    //
+    // Recenter the field to the nodes if necessary.
+    // FIX_ME: This code is never executed because VTKmAble returns
```

Comment:
did you mean to leave this here

Reply:
That comment is correct. This code should be fixed at some point and then VTKmAble needs to me modified to return true instead of false when it encounters cell centered variables.